content/en/docs/concepts/containers/images.md

```diff
@@ -375,4 +298,7 @@ common use cases and suggested solutions.
 If you need access to multiple registries, you can create one secret for each registry.
 Kubelet will merge any `imagePullSecrets` into a single virtual `.docker/config.json`
 
+## {{% heading "whatsnext" %}}
 
+* Read the [OCI Image Manifest Specification](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+{{% /capture %}}
```

Comment:
I think this is causing the build failure.

Reply:
Yep! Oops! Fixed!